### PR TITLE
Minor Changes for VEC Sample Code and some code formatting

### PR DESCRIPTION
**Pull request by bipul-agarwal**
1. Added deeplink handling code.
2. Some minor formatting changes

@rekrishn Please review and merge.

### CODE REVIEW

File: using-mobile-extensions/adobe-target-vec.md [hunk 1]
```diff
@@ -57,20 +60,21 @@ To get started with Target VEC, complete the following steps:
...
-2. Import the Target VEC extension in your application's main activity.  `import com.adobe.target.mobile.TargetVEC;`
-3. Add the Target VEC library to your project via your `Podfile` by adding `pod 'ACPTargetVEC'`
+2. Import the Target VEC extension in your application's main activity.  `import com.adobe.target.mobile.TargetVEC;
 
-#### Objective-C
+{% endtab %}
+
+{% tab title="iOS" %}
+
+1. Import Target and TargetVEC library.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target-vec.md]: Please add "the" between "and" and "TargetVEC".

File: using-mobile-extensions/adobe-target-vec.md [hunk 2]
```diff
@@ -135,21 +142,41 @@ public class SampleApp extends Application {
 }
 ```
 
+2. In your app's `openURL` method, add the deeplink handling code
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target-vec.md]: Please add a period at end of step 2.

File: using-mobile-extensions/adobe-target-vec.md [hunk 3]
```diff
@@ -135,21 +142,41 @@ public class SampleApp extends Application {
...
-  ACPTargetVEC.registerExtension()
   ACPTarget.registerExtension()
+  ACPTargetVEC.registerExtension()
   [ACPCore start:^{
     [ACPCore lifecycleStart:nil];
   }];
   return true
 }
 ```
+2. In your app's `open url` method, add the deeplink handling code
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target-vec.md]: Please add a period at the end of step 2.

**Comment by bipul-agarwal** [comment]
Thanks @rekrishn 
I have made the suggested changes.

Merging to master branch.

**Pull request merged**